cheminformatics/deploy/all_Covid_tools.yaml

```diff
@@ -0,0 +1,148 @@
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: true
+tools:
+- name: enumerate_charges
+  owner: bgruening
+  revisions:
+  - 2a868592ebcb
+  tool_panel_section_label: Covid-19
```

Comment:
Should we have a Cheminformatics category and a genomics one?

Reply:
Good idea, should covid-19 still be mentioned ?

Reply:
Your call, your baby :)